test/e2e/apps/job.go

```diff
@@ -45,6 +47,10 @@ var _ = SIGDescribe("Job", func() {
 	f := framework.NewDefaultFramework("job")
 	parallelism := int32(2)
 	completions := int32(4)
+
+	largeParallelism := int32(100)
```

Comment:
Would this be enough to reproduce in any cluster size?

Maybe you can use pod affinity to force all the pods into a single node.

Reply:
The test is already using nodeSelector to pin all of the pods to a single node (see line 394 - https://github.com/kubernetes/kubernetes/pull/108352/files#diff-909ebfa7fcb2a94030da3371d35863a2d723fc93356334e35f68816f5e313379R394)

Reply:
missed that. Thanks